<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pwbDW</class>
 <widget class="QDockWidget" name="pwbDW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>338</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::NoDockWidgetFeatures</set>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>P r:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TPBrUnitInput" name="pres" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text" stdset="0">
         <string>0</string>
        </property>
        <property name="unitType" stdset="0">
         <enum>EPressure</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>P wb:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TPBrUnitInput" name="pwb" native="true">
        <property name="text" stdset="0">
         <string>19.5</string>
        </property>
        <property name="unitType" stdset="0">
         <enum>EPressure</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Fluid Model:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="FluidModelRB_NP">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Non-penetrating</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="FluidModelRB_P">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Penetrating</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="FluidModelRB_Const">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Constant (Pres)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Steps:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="steps">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>72</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="runBtn">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>248</width>
        <height>9</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TPBrUnitInput</class>
   <extends>QWidget</extends>
   <header>tpbrunitinput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
